test/everypolitician/popolo/json_test.rb

```diff
@@ -26,4 +26,8 @@ def test_that_terms_returns_the_same_as_legislative_periods
   def test_current_term_returns_the_same_as_current_legislative_period
     assert_equal popolo_json.current_term, popolo_json.current_legislative_period
   end
+
+  def test_that_terms_returns_legislative_period_objects
+    assert_instance_of Everypolitician::Popolo::LegislativePeriod, popolo_json.terms.first
+  end
 end
```

Comment:
I think I'd want to be a bit more explicit here and test that _every_ element of `terms` is the correct object. I'd probably also want to test that everything that we know _is_ a term gets returned from here.